languages/java/exercises/concept/arrays/.meta/design.md

```diff
@@ -0,0 +1,37 @@
+Of the many available Java collection types, we chose to use the `array` collection type as the first collection type students will be taught for the following reasons:
+
+- Arrays don't require the student to know about generics.
+- Arrays are a common data type in many languages.
+- Arrays have a fixed length. No complexity in adding or removing elements.
+- Arrays have a simple shorthand syntax. No need to understand how constructors work to define an array.
+
+## Learning objectives
+
+- The existence of the `Array` type.
+- Defining an array.
+- Accessing elements in an array by index.
+- Updating an element in an array by index.
+- Iterating over elements in an array.
+- Basic array functions (like finding the index of an element in an array).
+
+## Out of scope
+
+- Multi-dimensional/jagged arrays.
+- Memory and performance characteristics of arrays.
+- Enumerables.
+- Iterators.
+- LINQ.
+
+## Concepts
+
+- `arrays`: know of the existence of the `Array` type; know how to define an array; know how to access elements in an array by index; know how to update an element in an array by index; know how to iterate over elements in an array; know of some basic functions (like finding the index of an element in an array).
+- `for-loops`: know how to use a `for` loop to do iteration.
+- `foreach-loops`: know how to iterate over a collection.
+
+## Prequisites
+
+This exercise's prerequisites Concepts are:
+
+- `classes`: know how to work with fields.
+- `booleans`: know what a `bool` is.
```

Comment:
When it is possible it is better to put the Java concept as keyword, so prefer `boolean` instead of `bool`